mex/extractors/wikidata/helpers.py

```diff
@@ -28,7 +28,7 @@ def get_wikidata_organization_by_id(wikidata_id: str) -> ExtractedOrganization |
         msg = "Primary source for wikidata not found"
         raise MExError(msg)
     extracted_organization = transform_wikidata_organization_to_extracted_organization(
-        wikidata_organization, wikidata_primary_source
+        wikidata_organization, wikidata_primary_source.stableTargetId
```

Comment:
just as an idea:
```suggestion
        wikidata_organization, get_extracted_primary_source_id_by_name("wikidata")
```
and delete lines 26-29